ci/lepton/model_convergence/scripts/wrap_template.sh

```
@@ -0,0 +1,271 @@
+    gpu_uuid = r[0].strip() if len(r)>0 else None
+    # pid as int where possible
+    pid = None
+    if len(r)>1:
+        try: pid = int(r[1].strip())
+        except: pid = None
+    process  = r[2].strip() if len(r)>2 else None
+    used_mem = r[3].strip() if len(r)>3 else None
+    rows.append({"gpu_uuid": gpu_uuid, "pid": pid, "process_name": process, "used_memory": used_mem})
+print(json.dumps(rows))
+PY
+    <<< "$RAW_APPS"
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)